common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -24,32 +26,43 @@ window.VerticalStudentView = function(runtime, element) {
             apiUrl: $bookmarkButtonElement.data('bookmarksApiUrl')
         });
     });
-    $(element).find('.vert').each(
-        function(idx, block) {
-            var blockKey = block.dataset.id;
-
-            if (block.dataset.completableByViewing === undefined) {
-                return;
-            }
-            // TODO: EDUCATOR-1778
-            // *  Check if blocks are in the browser's view window or in focus
-            //    before marking complete. This will include a configurable
-            //    delay so that blocks must be seen for a few seconds before
-            //    being marked complete, to prevent completion via rapid
-            //    scrolling.  (OC-3358)
-            // *  Limit network traffic by batching and throttling calls.
-            //    (OC-3090)
-            if (blockKey && !SEEN_COMPLETABLES.has(blockKey)) {
-                $.ajax({
-                    type: 'POST',
-                    url: runtime.handlerUrl(element, 'publish_completion'),
-                    data: JSON.stringify({
-                        block_key: blockKey,
-                        completion: 1.0
-                    })
-                });
-                SEEN_COMPLETABLES.add(blockKey);
+    RequireJS.require(['bundles/ViewedEvent'], function() {
```

Comment:
This is still weird, but eslint doesn't complain now.  Note that we are "requiring" the ViewedEvent bundle, but not actually providing a function parameter for it.